src/mame/drivers/missb2.cpp

```diff
@@ -518,7 +508,7 @@ ROM_START( missb2 )
 	ROM_LOAD( "msbub2-u.203", 0x10000, 0x10000, CRC(29fd8afe) SHA1(94ead80d20cd3974dd4fb0358915e3bd8b793158) )
 	/* 20000-2ffff empty */
 
-	ROM_REGION( 0x10000, "slave", 0 ) /* 64k for the second CPU */
+	ROM_REGION( 0x10000, "subcpu", 0 ) /* 64k for the second CPU */
```

Comment:
Was done in missb2.cpp to be consistent with the region names in bublbobl.cpp, which changed due to the schematics. Since missb2 inherits a lot of stuff from bublbobl.cpp, I wasn't sure if it was necessary or not to make the region names the same, so I did it out of abundance of caution.